keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+import argparse
+import pycparser
+
+def beautify_src_file(filename):
+    src = {
+        "before": [],
+        "keys": [],
+        "after": []
```

Comment:
```suggestion
        "after": [],
```
Trailing commas are a best practice suggested by many good style guides.